app/javascript/mastodon/features/__tests__/toggle-play-test.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
```

Comment:
Aside: I don't think you need the name "test" in the file

Reply:
@nschonni Nice, just commited the change now.